modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml

```diff
@@ -179,7 +246,7 @@ components:
           type: array
           items:
             type: string
-    EsenderAuthor:
+    NoticeAuthor:
```

Comment:
The current specification defines "EsenderAuthor", not "NoticeAuthor".

Reply:
I think Vangelis is right for this one. You are changing a schema. You can't